src/components/SearchBar/index.jsx

```diff
@@ -48,6 +64,8 @@ function SearchBar() {
       </DesktopSearchBarContainer>
       <MobileSearchBarContainer>
         <MobileSearchBar
+          aria-label="Mobile Search Jobs"
+          aria-labelledby="mobile-search-bar-label"
```

Comment:
Love your inclusion of aria attributes for accessibility purposes ❤️

Reply:
Thank you!